### ISSUE TITLE
Custom shortcuts

### ISSUE THREAD

**Issue by wiruzx** [kolinkrewinkel/Multiplex#16]
Hi Kolin,
First of all, I'd like to say thanks for the plugin, really great work!
But it would be really nice, if the plugin didn't override default Xcode's shortcuts, like `CMD-click`.
Is it possible to allow users to chose their own shortcuts for actions?

**Comment by kolinkrewinkel** [comment]
Yeah, I'd like to have it be this way...I'll need to think about if I want to integrate it into the existing preferences view for this.

It poses a challenge because the long-click isn't really a gesture you could have in the existing settings. Thus, I think a separate settings view that adjusts the source editor's "text interactions/shortcuts" (add new cursor, jump to definition, show info popover) might be the way.

**Comment by iamcam** [comment]
Would ⌥ + ⌘ + Click be another option? I also would like to maintain the default Xcode shortcuts.
